ocs_ingester/fits.py

```diff
@@ -98,6 +107,51 @@ def set_public_date(self):
                     parse(self.fits_dict['DATE-OBS']) + timedelta(days=365)
                 ).isoformat()
 
+    def truncate_exptime(self):
```

Comment:
Minor, but maybe name it `round_exptime`, since that's technically what you're doing?

Reply:
done